### PR TITLE
Update session details

### PR DESCRIPTION
**Pull request by priyanka-surana**
Updating the material covered in each session. I have added my best guess, can you please update and merge.

### CODE REVIEW

File: src/content/events/2023/training-sept-2023/wgc.md [hunk 1]
```diff
@@ -41,30 +41,42 @@ The training material will be presented virtually by Seqera Labs bioinformatics
             <tbody>
             <tr>
                 <td >9:30 AM</td>
-                <td background-color:navy; rowspan="1">Session 1</td>
+                <td background-color:navy; rowspan="1">
```
Comment (mashehu) [inline src/content/events/2023/training-sept-2023/wgc.md:44]:
```suggestion
                <td>
```
This doesn't do anything (same for all the other `<td>`s

**Review by mashehu — APPROVED** [review]

File: src/content/events/2023/training-sept-2023/wgc.md (hunk 1 above)
Comment (priyanka-surana) [reply src/content/events/2023/training-sept-2023/wgc.md]: Fixed all instances.

**Comment by christopher-hakkaart** [comment]
It might be a good idea to add a note that these titles might differ slightly on the day. Just to give some wiggle room incase things change on the day.

**Comment by priyanka-surana** [comment]
> It might be a good idea to add a note that these titles might differ slightly on the day. Just to give some wiggle room incase things change on the day.

Added something. But feel free to make any changes.

**Review by christopher-hakkaart — APPROVED** [review]

**Pull request merged**